<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="fgdc_citeinfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>220</height>
      </size>
     </property>
     <property name="title">
      <string>Citation</string>
     </property>
     <widget class="QLineEdit" name="fgdc_title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>901</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Data Set Title</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>731</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>A good title includes 'What', 'Where', and 'When'.  (Example: Point Locations of Wind Turbines in Colorado, Derived from 2010 NAIP Imagery</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Data Set Author / Originator</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Who created the data set? List the organization and/or person(s)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Publication Date (YYYYMMDD)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>110</y>
        <width>261</width>
        <height>200</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="text">
       <string>When was the data set published or finalized?</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fgdc_origin">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>361</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="fgdc_pubdate">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>130</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Is this data set associated with a larger work?</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>0</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>180</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>If the citation of a larger project is relevant, you may optionally cite it here.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>90</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Online Link for the Data Set</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>130</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>110</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Is there a link to the data or agency that produced it?</string>
      </property>
     </widget>
     <zorder>fgdc_title</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>fgdc_origin</zorder>
     <zorder>fgdc_pubdate</zorder>
     <zorder>label_7</zorder>
     <zorder>groupBox</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>radioButton</zorder>
     <zorder>lineEdit</zorder>
     <zorder>label_10</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
